Python/Product/VSInterpreters/InterpreterOptionsService.cs

```diff
@@ -226,7 +226,7 @@ sealed class InterpreterOptionsService : IInterpreterOptionsService {
             using (var key = Registry.CurrentUser.CreateSubKey(collection, true)) {
                 key.SetValue(LibraryPathKey, config.LibraryPath ?? string.Empty);
                 key.SetValue(ArchitectureKey, config.ArchitectureString);
-                key.SetValue(VersionKey, config.Version.ToString());
+                key.SetValue(VersionKey, config.Version?.ToString() ?? "2.7");
```

Comment:
Do we really want to keep using "2.7" as the default? Do we have any back-compat dependencies on this?